Markup kadai 06

blue2wo opened the pull request:
練習課題6「ブロックを横に並べてみよう」1st commit
16cbf58 HTML CSS

piasoratech commented on markup_kadai_06/markup_kadai_06.html:
```diff
@@ -0,0 +1,15 @@
+        <div class="item" id="square2"></div>
+    </div>
+    
+<!--    <div id="square3"></div>-->
```
不要なコードは含めないようにしましょう！

piasoratech commented on markup_kadai_06/styleE6.css:
```diff
@@ -0,0 +1,22 @@
+body {
+    background-color: peachpuff;
```
色は名前で指定するよりカラーコードで指定するほうが一般的なので、カラータグで指定してみましょう！

blue2wo commented:
9666aa0 
- removed unnecessary code and other kadai files. Only kadai 06 files remain.
- removed `background-color: peachpuff;`, example change `background-color: #818488`

piasoratech commented on markup_kadai_06/markup_kadai_06.html:
```diff
@@ -0,0 +1,13 @@
+</head>
+<body>
+    <div class="container">
+        <div class="item" id="square1"></div>
+        <div class="item" id="square2"></div>
```
`id`や`class`名にxxx1といったようなマジックナンバーを使用するのはやめたほうが良いです。

piasoratech commented on markup_kadai_06/styleE6.css:
```diff
@@ -0,0 +1,18 @@
+/*    以下は必要ない*/
+}
+
+#square1 {
```
`id`や`class`名にxxx1といったようなマジックナンバーを使用するのはやめたほうが良いです。

piasoratech commented on markup_kadai_06/styleE6.css:
```diff
@@ -0,0 +1,18 @@
+    background-color: #FF0000;
+}
+
+#square2 {
```
`id`や`class`名にxxx1といったようなマジックナンバーを使用するのはやめたほうが良いです。

blue2wo commented:
練習課題6「ブロックを横に並べてみよう」3rd push - feedback changes

2e17db6
- changed id names to be more suitable #square1 to #squarered

fjtawork commented on markup_kadai_06/styleE6.css:
```diff
@@ -0,0 +1,18 @@
+    display: flex;
+    width: 550px;
+    justify-content: space-between;
+/*    以下は必要ない*/
```
いらないコメントは消しましょう！
